BadHabbit/BadHabbit/SupportFiles/Fonts/Fonts.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Fonts.swift
```

Comment:
I don't see this file
<img width="238" alt="Screenshot 2023-08-11 at 18 19 22" src="https://github.com/The-Lessoner/bad_habbit/assets/97001077/6bd51d43-6a51-42c4-9b4f-638aff67df3a">

Reply:
resolved